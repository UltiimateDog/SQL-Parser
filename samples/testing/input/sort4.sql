SELECT Student.*, Enrolled.E, Enrolled.H FROM Student, Enrolled ORDER BY Student.B, Enrolled.E;