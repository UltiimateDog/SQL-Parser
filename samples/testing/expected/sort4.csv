3,100,105,44,101,75
3,100,105,44,101,12
4,100,50,11,101,75
4,100,50,11,101,12
5,100,500,22,101,75
5,100,500,22,101,12
3,100,105,44,102,82
3,100,105,44,102,52
4,100,50,11,102,82
4,100,50,11,102,52
5,100,500,22,102,82
5,100,500,22,102,52
3,100,105,44,103,92
4,100,50,11,103,92
5,100,500,22,103,92
3,100,105,44,104,27
4,100,50,11,104,27
5,100,500,22,104,27
1,200,50,33,101,75
1,200,50,33,101,12
2,200,200,44,101,75
2,200,200,44,101,12
1,200,50,33,102,82
1,200,50,33,102,52
2,200,200,44,102,82
2,200,200,44,102,52
1,200,50,33,103,92
2,200,200,44,103,92
1,200,50,33,104,27
2,200,200,44,104,27
6,300,400,11,101,75
6,300,400,11,101,12
6,300,400,11,102,82
6,300,400,11,102,52
6,300,400,11,103,92
6,300,400,11,104,27
